### ISSUE TITLE
Expression of type 'Union[str, bytes]' cannot be assigned to declared type 'str'

### ISSUE THREAD

**Issue by starship863** [Microsoft/pyright#103]
**Describe the bug**
Expression of type 'Union[str, bytes]' cannot be assigned to declared type 'str'
 Type 'bytes' cannot be assigned to type 'str'.
  'Type[bytes]' is incompatible with 'Type[str]'

**To Reproduce**
```
parent_path: str = os.path.dirname(sys.path[0])
if parent_path not in sys.path:
    sys.path.append(parent_path)
```

**Expected behavior**
No problems reported

**Screenshots or Code**
![image](https://user-images.githubusercontent.com/36103976/56952733-f0fd7980-6b75-11e9-8f23-d11a2aa6811e.png)

**VS Code extension or command-line**
I am running pyright as a VS Code extension 1.0.19

**Additional context**
Thank you so much for your great work!

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in the next version.

**Comment by erictraut** [comment]
This is now fixed in 1.0.20, which I just published.
